### PR TITLE
Added skipPrepare 

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Added skipPrepare for some scripts that are no needs to preparation fo incident to alert.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Securonix/Scripts/SecuronixCloseHistoricalXSOARIncidents/SecuronixCloseHistoricalXSOARIncidents.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ name: SecuronixCloseHistoricalXSOARIncidents
 script: ''
 type: python
 tags: []
+skipPrepare:
```
Comment (yuvalbenshalom) [inline Packs/Securonix/Scripts/SecuronixCloseHistoricalXSOARIncidents/SecuronixCloseHistoricalXSOARIncidents.yml:8]: should not be skipped

File: Packs/SumoLogic_Cloud_SIEM/Scripts/SumoLogicLinkSignalIncidents/SumoLogicLinkSignalIncidents.yml [hunk 2]
```diff
@@ -15,6 +15,8 @@ script: ''
 scripttarget: 0
 subtype: python3
 tags: []
+skipPrepare:
+- script-name-incident-to-alert
```
Comment (yuvalbenshalom) [inline Packs/SumoLogic_Cloud_SIEM/Scripts/SumoLogicLinkSignalIncidents/SumoLogicLinkSignalIncidents.yml:19]: do not skip

File: Packs/qualys/Scripts/QualysCreateIncidentFromReport/QualysCreateIncidentFromReport.yml [hunk 3]
```diff
@@ -7,6 +7,8 @@ type: python
 subtype: python3
 tags:
 - qualys
+skipPrepare:
+- script-name-incident-to-alert
```
Comment (yuvalbenshalom) [inline Packs/qualys/Scripts/QualysCreateIncidentFromReport/QualysCreateIncidentFromReport.yml:11]: remove skip

File: Packs/qualys/pack_metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "name": "Qualys",
     "description": "Qualys Vulnerability Management let's you create, run, fetch and manage reports, launch and manage vulnerability and compliance scans, and manage the host assets you want to scan for vulnerabilities and compliance",
     "support": "xsoar",
-    "currentVersion": "1.2.6",
+    "currentVersion": "1.2.7",
```
Comment (yuvalbenshalom) [inline Packs/qualys/pack_metadata.json:5]: remove

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
see my comments.
also lets consider making the release notes specific for scripts.

**Comment by dorschw** [comment]
@yuvalbenshalom please force-merge this one. The only failing validations are `DO106`, `RM109`, `CJ105`.

**Pull request merged**